rouge-score 0.1.2 :snowflake:

ifitchet opened the pull request:
rouge-score 0.1.2 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-4823]
- dev_url:        https://github.com/google-research/google-research/tree/master/rouge
- conda_forge:    https://github.com/conda-forge/rouge-score-feedstock/blob/main/recipe/meta.yaml
- pypi:           https://pypi.org/project/rouge-score/0.1.2
- pypi inspector: https://inspector.pypi.io/project/rouge-score/0.1.2

### Explanation of changes:

- clone of conda-forge recipe
- > [!WARNING] no upstream tests, the upstream is 1.3GB of monorepo with no tags or tarballs thus no sane way to get the testdata to see if it works
- copy individual testdata file from GH
- skip tests on Windows (re-use of temporary files)


[PKG-4823]: https://anaconda.atlassian.net/browse/PKG-4823?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

ifitchet committed:
abs.yaml

ifitchet committed:
basic build

anaconda-pkg-build commented:
<details><summary>Linter check found the following problems:</summary>
ERROR conda.cli.main_run:execute(125): `conda run conda-lint /tmp/abs_410r6vdbfv/clone` failed. (See above for error)
The following problems have been found:

===== ERRORS =====
- clone/recipe/meta.yaml:68: missing_license_family: The recipe is missing the `about/license_family` key.
- clone/recipe/meta.yaml:20: missing_wheel: For pypi packages, wheel should be present in the host section
- clone/recipe/meta.yaml:68: missing_dev_url: The recipe is missing a dev_url
- clone/recipe/meta.yaml:68: missing_documentation: The recipe is missing a doc_url or doc_source_url
===== Final Report: =====
4 Errors and 0 Warnings were found

</details>

ifitchet committed:
post-conda lint:

There's a distinct shortage of associated docs, homepage etc..
Basically everything boils down to the dev_url.

ifitchet committed:
add upstream tests:

This pulls three files from the 1.3GB upstream monorepo to allow us to
run at least some tests.  We could do a few more if we could
accommodate the punkt thing.

ifitchet committed:
skip io_test on Windows

It looks like there is re-use of temporary files on Windows which
generates the Permission denied errors.

ViridianMelody commented on recipe/meta.yaml:
```diff
@@ -67,6 +112,10 @@ about:
     PyPI: [https://pypi.org/project/rouge-score/](https://pypi.org/project/rouge-score/)
 
+  # There's no clear doc_url other than this README
+  doc_url: https://github.com/google-research/google-research/tree/master/rouge
```
```suggestion
  doc_url: https://github.com/google-research/google-research/blob/master/rouge/README.md
```

ifitchet replied:
Great spot!

ViridianMelody reviewed (approved):
Well done getting some of the tests running! Minor suggestion to link directly to the readme for doc_url

skupr-anaconda reviewed (approved):
LGTM but Melody's comment

ifitchet committed:
reference the README (not the tree)

Co-authored-by: Melody Love <mlove@anaconda.com>
